src/services/ApiService/Faucet/types.ts

```diff
@@ -15,7 +15,7 @@ export interface FaucetChallengeResponse {
   };
 }
 
-export interface FaucetSolvedChallengeResponse {
+interface FaucetSolvedChallengeResponseV1 {
```

Comment:
Is there a reason to keep the old version around?

Reply:
@FrederikBolding If we only include V2 the faucet will break until the server is upgraded. We could probably remove it after that.

Reply:
Cool! Please make a card to do so 👍